cl-dot.lisp

```diff
@@ -185,10 +185,11 @@ from OBJECTS, using the GRAPH-OBJECT- protocol.")
 (defun dot-graph (graph outfile &key (format :ps) (directed t))
   "Renders GRAPH to OUTFILE by running the program in \*DOT-PATH* or
 *NEATO-PATH* depending on the value of the DIRECTED keyword
-argument.  The default is a directed graph.  The default
-FORMAT is Postscript."
+argument.  The default is a directed graph. 
+The output file is assigned a suffix based on the FORMAT.
+The default FORMAT is Postscript."
   (when (null format) (setf format :ps))
```

Comment:
This line is unnecessary.  Unless the caller explicitly writes `(dot-graph foo "/my/out/file.xxx" :format nil)`, any failure to specify a format will get the default value.